src/components/PhoneNumberAuth/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react'
+import firebase, { firebaseErrorResponse } from '../../firebase'
+
+
+function PhoneNumberSuccess(props) {
+  return (
+    <div>
+      Success
+    </div>
+  )
+}
+
+export default function(props) {
+    const [phoneNumber, setPhoneNumber] = useState('')
+    const [confirmationResult, setConfirmationResult] = useState(null)
+    const [code, setCode] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const [user, setUser] = useState(null)
+    const [loading, setLoading] = useState(false)
```

Comment:
Maybe replace with suspense?
https://reactjs.org/docs/code-splitting.html#suspense

Reply:
Yeah Let me check how that can work out, good point!

Reply:
Will probably make an other PR out of this so we can get this merged